### PR TITLE
Functionality of applying Show Intent on Whole Data Added

### PR DESCRIPTION
**Pull request by anmoliiitd823**
Files changed : show.py and test_show.py
- Added the functionality of applying summary operation on whole data
- Added tests in test_show.py to test this feature

### CODE REVIEW

File: intents/show.py [hunk 1]
```diff
@@ -71,34 +71,85 @@ def show(table,**kwargs):
...
+        dimensions.append('Summary Operator')
+
+    table = aspects.group_by(table, dimensions, summary_operator)
+
+    return table
+
+
+
+
+def add_temporary_column_of_summary_operator(table,summary_operator):
```
Comment (arijitde-goog) [inline intents/show.py:120]: Since this is an internal function, prefix name with _

Comment (anmoliiitd823) [reply intents/show.py]: Updated

File: intents/show.py [hunk 2]
```diff
@@ -71,34 +71,85 @@ def show(table,**kwargs):
...
+    summary_operator = kwargs.get('summary_operator', None)
 
-	dimensions = kwargs.get('dimensions',None)
+    metric = kwargs.get('metric',None)
 
-	table = aspects.apply_date_range(table, date_range,date_column_name, date_format)
+    dimensions = kwargs.get('dimensions',None)
 
-	table = aspects.slice_table(table, slices)
+    table = aspects.apply_date_range(table, date_range,date_column_name, date_format)
```
[the remark this answers is not in the document]
Comment (anmoliiitd823) [reply intents/show.py]: Done

File: intents/show.py [hunk 3]
```diff
@@ -71,34 +71,85 @@ def show(table,**kwargs):
...
+    table = aspects.crop_other_columns(table, required_columns)
 
-	return table
-    
+    # When there is no dimension to group by but a summary_operator exists ,
+    # we assume that user wants to apply the summary operator on whole data
+    if( (dimensions is None) and (summary_operator is not None) ):
+        # We add temporary column of 'Summary Operator' by which we can groupby
+        # to obtain the final result
+        add_temporary_column_of_summary_operator(table,summary_operator)
```
[the remark this answers is not in the document]
Comment (anmoliiitd823) [reply intents/show.py]: Done

**Pull request merged**